pyg_lib/csrc/random/cpu/biased_sampling.cpp

```diff
@@ -0,0 +1,60 @@
+#include "pyg_lib/csrc/random/cpu/biased_sampling.h"
+
+namespace pyg {
+namespace random {
+
+at::Tensor biased_to_cdf(at::Tensor rowptr, at::Tensor bias) {
+  TORCH_CHECK(rowptr.is_cpu(), "'rowptr' must be a CPU tensor");
+  TORCH_CHECK(bias.is_cpu(), "'bias' must be a CPU tensor");
+
+  auto cdf = at::empty_like(bias);
```

Comment:
```suggestion
  const auto cdf = at::empty_like(bias);
```

Reply:
We need to write data to cdf so it's not a const here

Reply:
The tensor will always be a const since we only change its data, not its metadata. It‘s not so important though :)